src/PrestaShopBundle/Resources/views/Admin/layout.html.twig

```diff
@@ -21,23 +21,59 @@
  * @author    PrestaShop SA and Contributors <contact@prestashop.com>
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
- *#}
-{% extends(template_from_string(
-  getLegacyLayout(
-    app.request.attributes.get('_legacy_controller'),
-    layoutTitle is defined ? layoutTitle : '',
-    layoutHeaderToolbarBtn is defined ? layoutHeaderToolbarBtn : [],
-    layoutDisplayType is defined ? layoutDisplayType : '',
-    showContentHeader is defined ? showContentHeader : true,
-    headerTabContent is defined ? headerTabContent : '',
-    enableSidebar is defined ? enableSidebar : false,
-    help_link is defined ? help_link : '',
-    js_router_metadata(),
-    meta_title is defined ? meta_title : '',
-    use_regular_h1_structure is defined ? use_regular_h1_structure : true,
-    legacyBaseLayout is defined ? legacyBaseLayout : 'layout.tpl'
+ * #}
+{% set lightDisplay = lightDisplay is defined ? lightDisplay : app.request.query.get('liteDisplaying', false)  %}
+{% set showContentHeader = showContentHeader is defined ? showContentHeader : true  %}
+{% set layoutHeaderToolbarBtn = layoutHeaderToolbarBtn is defined ? layoutHeaderToolbarBtn : [] %}
+{% set metaTitle = meta_title is defined ? meta_title : (layoutTitle is defined ? layoutTitle : '') %}
+
+{% extends(use_symfony_layout
+  ? lightDisplay ? '@PrestaShop/Admin/Layout/light_layout.html.twig' : '@PrestaShop/Admin/Layout/default_layout.html.twig'
+  : template_from_string(
+    getLegacyLayout(
+      app.request.attributes.get('_legacy_controller'),
+      layoutTitle is defined ? layoutTitle : '',
+      layoutHeaderToolbarBtn,
+      layoutDisplayType is defined ? layoutDisplayType : '',
+      showContentHeader,
+      headerTabContent is defined ? headerTabContent : '',
+      enableSidebar is defined ? enableSidebar : false,
+      help_link is defined ? help_link : '',
+      js_router_metadata(),
+      meta_title is defined ? meta_title : '',
+      use_regular_h1_structure is defined ? use_regular_h1_structure : true,
+      legacyBaseLayout is defined ? legacyBaseLayout : 'layout.tpl'
+    )
   )
-  )) %}
+) %}
+
+{% block coreJavascripts %}
+  <script src="{{ asset('themes/new-theme/public/main.bundle.js') }}"></script>
```

Comment:
I'm a bit concerned about adding this block here 🤔 It was not present previously meaning it has no usage for a page that will rely on legacy layout so in these cases it's at best a dead useless block, at worst it can have unexpected side effects.

Now I know with twig heritage system we can't do everything we ant, but isn't it possible to define this block in `default_layout.html.twig` instead?

The less we modify this layout template the less chance we have of introducing unexpected side effects